scripts/linters/pre_commit_linter.py

```diff
@@ -230,13 +230,15 @@ def _lint_all_files(
     css_file_extension_type = 'css' in file_extensions_to_lint
     other_file_extension_type = 'other' in file_extensions_to_lint
 
-    if not other_file_extension_type:
-        general_files_to_lint = '.' + file_extensions_to_lint
+    if js_ts_file_extension_type:
+        general_files_to_lint = _FILES['.js'] + _FILES['.ts']
```

Comment:
Do we have any integration test of pre_commit_linter with the general_purpose linter script?

Reply:
We do not have integration tests yet.